src/eos-updater-avahi.c

```diff
@@ -629,17 +673,28 @@ generate_v2_service_file (OstreeRepo *repo,
                                                   error);
 }
 
+static gchar *
+get_avahi_services_dir (void)
+{
+  return eos_updater_dup_envvar_or ("EOS_UPDATER_TEST_UPDATER_AVAHI_SERVICES_DIR",
+                                    "/etc/avahi/services");
+}
+
 gboolean
 eos_avahi_generate_service_file (OstreeRepo *repo,
                                  EosBranchFile *branch_file,
                                  GError **error)
 {
   g_autoptr(GFile) service_file = NULL;
+  g_autofree gchar *services_dir = NULL;
+  g_autofree gchar *service_file_path = NULL;
 
   g_return_val_if_fail (OSTREE_IS_REPO (repo), FALSE);
   g_return_val_if_fail (EOS_IS_BRANCH_FILE (branch_file), FALSE);
 
-  service_file = g_file_new_for_path ("/etc/avahi/services/eos-updater.service");
+  services_dir = get_avahi_services_dir ();
+  service_file_path = g_build_filename (services_dir, "eos-updater.service", NULL);
+  service_file = g_file_new_for_path (service_file_path);
```

Comment:
`services_dir` in line below, `service_file_path` in another line below and `service_file` is passed to `generate_vX_service_file`. So yes, they are.